src/main/java/com/google/sps/data/Session.java

```diff
@@ -0,0 +1,53 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+public final class Session {
```

Comment:
I'd suggest naming this something different as "Session" has a specific meaning in terms of HTTP and web which might be confusing here.

Maybe "TutorSession" or "Event"? Also please add javadocs as described here:
https://google.github.io/styleguide/javaguide.html#s7.3-javadoc-where-required

Reply:
That makes a lot of sense. I pushed new commits to this PR that address that.